src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -1090,22 +1093,28 @@ static enum e_op_result execute_stream_op(struct op_and_state* oas) {
       grpc_slice_buffer write_slice_buffer;
       grpc_slice slice;
       grpc_slice_buffer_init(&write_slice_buffer);
-      if (1 != stream_op->payload->send_message.send_message->Next(
-                   stream_op->payload->send_message.send_message->length(),
-                   nullptr)) {
-        /* Should never reach here */
-        GPR_ASSERT(false);
-      }
-      if (GRPC_ERROR_NONE !=
-          stream_op->payload->send_message.send_message->Pull(&slice)) {
-        /* Should never reach here */
-        GPR_ASSERT(false);
+      while (write_slice_buffer.length <
+             stream_op->payload->send_message.send_message->length()) {
+        /* TODO(roth): When we add support for incremental sending,this code
+         * will need to be changed to support asynchronous delivery of the
+         * send_message payload. */
+        if (1 != stream_op->payload->send_message.send_message->Next(
```

Comment:
nit:  the return type of `Next()` is `bool`, so the `1 != ` part can be simply `!`

Reply:
done